infrastructure/templates/template.json

```diff
@@ -3037,12 +3012,12 @@
             {
               "name": "StatisticsDb",
               "type": "SQLAzure",
-              "connectionString": "[concat('Data Source=tcp:', reference(concat('Microsoft.Sql/servers/', variables('sqlServerName'))).fullyQualifiedDomainName, ',1433;Initial Catalog=', 'statistics', ';User Id=', parameters('sqlImporterUser'), '@', reference(concat('Microsoft.Sql/servers/', variables('sqlServerName'))).fullyQualifiedDomainName, ';Password=', parameters('sqlImporterUserPassword'), ';')]"
+              "connectionString": "[concat('Data Source=tcp:', reference(concat('Microsoft.Sql/servers/', variables('coreSqlServerName'))).fullyQualifiedDomainName, ',1433;Initial Catalog=', variables('coreStatisticsDbName'), ';User Id=', parameters('sqlImporterUser'), '@', reference(concat('Microsoft.Sql/servers/', variables('coreSqlServerName'))).fullyQualifiedDomainName, ';Password=', parameters('sqlImporterUserPassword'), ';')]"
```

Comment:
I think it would be worth adding EnableRetryOnFailure to all of these as we've seen one or two connection failures recently.

Reply:
Happy to try this - would it be OK to do it separately from this work though?

Reply:
Yep, just realised I don't think it's in the connection, but in code so we wouldn't  do it here.
I've raised [EES-3779](https://dfedigital.atlassian.net/browse/EES-3779) thanks.